src/Native/System.Private.CoreLib.Native/pal_errno.cpp

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include <errno.h>
+
+extern "C" int CoreLibNative_GetLastErrNo()
```

Comment:
This shim APIs should use explicitly sized types. From https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/interop-guidelines.md:

Use int32_t, int64_t, etc. from stdint.h and not int, long, etc